ui-v2/tests/steps.js

```diff
@@ -38,14 +38,14 @@ export default function(assert) {
       }, yadda)
     )
       // doubles
-      .given(['$number $model model', '$number $model models'], function(number, model) {
+      .given(['$number $model model[s]?', '$number $model models'], function(number, model) {
```

Comment:
Are these brackets special Yadda magic that makes the `s` optional? Does it work for other pluralizations (e.g., policy/policies)?

Reply:
Yup, other BDD stuff I've used in the past support this also, so not sure if its part of Gherkin to support this or not. But yeah its basically regex-y. My steps file is also prime area for refactoring shortly, just waiting for the right time.

> Does it work for other pluralizations (e.g., policy/policies)?

If you can come up with a regex for it you should be able to do it.

Think this is the docs:

https://acuminous.gitbooks.io/yadda-user-guide/en/usage/step-libraries.html#regular-expressions